resources/articles/oauth/token.md

```diff
@@ -0,0 +1,199 @@
+:title POST /oauth/token
+
+:aside
+
+## See also
+
+* [POST /oauth/ro](/oauth/ro/)
+* [POST /oauth/introspect](/oauth/introspect/)
+* [GET /oauth/userinfo](/oauth/userinfo/)
+* [GET /oauth/jwks](/oauth/jwks/)
+* [GET /oauth/authorize](/oauth/authorize/)
+* [SPiD Authentication](/authentication/)
+
+:body
+
+The `/oauth/token` endpoint is used to issue tokens to users and services.
+Users are given user tokens and services are given client/server tokens. In SPiD
```

Comment:
"Users are given user tokens and services are given client/server tokens." -> '"User tokens" are issued for users, while "client tokens" are issued for services.'